components/webui/imports/ui/SearchView/SearchResults/SearchResultsTimeline/index.jsx

```diff
@@ -262,7 +263,11 @@ const SearchResultsTimeline = ({
         <Bar
             className={"timeline-chart"}
             data={data}
-            options={options}/>
+            options={options}
+
+            // Deselect all selections to address a bug that prevents dragging
+            // on the chart when elements such as text or images are selected.
```

Comment:
```suggestion
            // If the user inadvertently selected the timeline, then dragging on it will drag the
            // timeline object rather than selecting a time range within it. Thus, on mouse down, we
            // clear any existing selections so that the following drag selects a time range.
```